### PR TITLE
Add DMARC record and introduce groups in the records

### PR DESCRIPTION
**Pull request by puneet60**

### CODE REVIEW

File: brevo.com.domain-authentication.json [hunk 1]
```diff
@@ -3,24 +3,41 @@
...
-        "data": "%dkim_public_key%",
-        "ttl": 3600
+      "groupId": "dkim_raw",
+      "type": "TXT",
+      "host": "%dkim_selector%._domainkey",
+      "data": "%dkim_public_key%",
+      "ttl": 3600
+    },
+    {
+      "groupId": "dmarc",
```
Comment (pawel-kow) [inline brevo.com.domain-authentication.json:28]: As DMARC tends to rise a need to be edited afterwards manually there is quite useful to define this record as non essential. Just for consideration.

File: brevo.com.domain-authentication.json [hunk 2]
```diff
@@ -3,24 +3,41 @@
...
   "records": [
     {
-        "type": "TXT",
-        "host": "@",
-        "data": "brevo-code:%hash%",
-        "ttl": 3600
+      "groupId": "code",
+      "type": "TXT",
+      "host": "@",
+      "data": "%hash%",
```
Comment (pawel-kow) [inline brevo.com.domain-authentication.json:17]: If the semantics of a variable changes I suggest to use a new name. This makes the rollover of the template easier when some providers still have the old version and maybe do not report the version over api. Pushing both variables new and old in the call does not pose a problem.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: brevo.com.domain-authentication.json (hunk 2 above)
Comment (puneet60) [reply brevo.com.domain-authentication.json]: Done

File: brevo.com.domain-authentication.json (hunk 1 above)
Comment (puneet60) [reply brevo.com.domain-authentication.json:28]: Thanks for suggestion. Done

**Review by pawel-kow — APPROVED** [review]
lgtm

**Pull request merged**